### ISSUE TITLE
tcp-line-server disconnected without any error message output in log file

### ISSUE THREAD

**Issue by 362115815** [questdb/questdb#2197]
### Describe the bug

I'm using ILP to insert data,  but the tcp-line-server always disconnect after sending several rows. I couldn't find some error message in log file,  so I printed out the row data and finally found that there is a space char in a symbol column value, eg:
'test,name=sss ,ind=ddd value=13\n' , the right format should be 'test,name=sss\ ,ind=ddd value=13\n'

the log writer didn't ouput any error message when tcp-line-server received an invalid line.

content in log file:
2022-06-08T08:30:31.560700Z I tcp-line-server connected [ip=192.168.50.134, fd=8]
2022-06-08T08:30:48.182772Z I tcp-line-server scheduling disconnect [fd=8, reason=0]
2022-06-08T08:30:48.182843Z I tcp-line-server disconnected [ip=192.168.50.134, fd=8, src=queue]

My log.conf :
# list of configured writers
writers=file
# file writer
w.file.class=io.questdb.log.LogFileWriter
w.file.location=questdb-docker.log
w.file.level=INFO,ERROR

### Environment

```markdown
- **QuestDB version**:
- **OS**:
- **Browser**:
```

**Comment by ideoma** [comment]
We have built few ILP client for different platforms, they check validity of the fields. 

https://github.com/questdb/net-questdb-client
https://github.com/questdb/c-questdb-client
https://github.com/questdb/go-questdb-client

Re the original issue, log can swallow some line under pressure. Was the server logging a lot the moment the disconnect happened?

**Comment by 362115815** [comment]
> 



> We have built few ILP client for different platforms, they check validity of the fields.
> 
> https://github.com/questdb/net-questdb-client https://github.com/questdb/c-questdb-client https://github.com/questdb/go-questdb-client
> 
> Re the original issue, log can swallow some line under pressure. Was the server logging a lot the moment the disconnect happened?

Nope, I sent only one line manually in python env through the socket and the disconnect happened. Not tested in other platforms. And other invalid case like an integer column received a float value,  the server would disconnect with an error msg:
```
2022-06-20T01:18:36.421256Z E i.q.c.l.t.LineTcpConnectionContext [755] could not process line data [table=test, msg=cast error for line protocol float [columnIndex=0, columnType=INT], errno=0]
2022-06-20T01:18:36.421326Z I tcp-line-server scheduling disconnect [fd=755, reason=0]
2022-06-20T01:18:36.421338Z I tcp-line-server disconnected [ip=192.168.50.134, fd=755, src=queue]
```
